source/javascripts/services/_appService.js.erb

```diff
@@ -115,6 +115,14 @@ angular.module("BitriseWorkflowEditor").service("appService", function($q, reque
 				}
 			}
 
+			if (shouldValuesBeNotEmpty) {
```

Comment:
Rather do a `Variable.isValidValue(value, shouldBeNotEmpty)` function, and use it in the same way as `Variable.isValidKey` above. And then add the related error the same way as above `if (shouldKeysBeUnique) {` by checking `shouldValuesBeNotEmpty`.